src/data/set/intervals/basic.lean

```diff
@@ -1153,6 +1153,70 @@ end
 
 end linear_order
 
+/-! ### Lemmas about membership of arithmetic operations -/
+
+section ordered_comm_group
+
+variables {α : Type*} [ordered_comm_group α] {a b c d : α}
+
+-- inv_mem_Ixx_iff, sub_mem_Ixx_iff
```

Comment:
We may as well make these visible on the web docs, what do you think?
```suggestion
/-! `inv_mem_Ixx_iff`, `sub_mem_Ixx_iff` -/
```

Reply:
No objection from me